src/components/auth/ApiKeyAuth/ApiKeyAuthContent.tsx

```diff
@@ -26,6 +26,7 @@ type ApiKeyAuthFormProps = {
   isButtonDisabled?: boolean;
   buttonVariant?: "ghost";
   submitButtonType?: "submit" | "button";
+  metadataFields: MetadataItemInput[];
```

Comment:
can you rename this (and every other prop that's currently `metadataFields`) to `metadataInputs` ? Fields should be reserved to mean SaaS data fields.

Reply:
I'll follow up with a PR for this